src/client/content_cao.h

```diff
@@ -208,6 +210,11 @@ class GenericCAO : public ClientActiveObject
 		return m_is_local_player;
 	}
 
+	inline bool isPlayer() const
```

Comment:
What is this change needed for?

Reply:
Oh I missed this when I fixed the player name behavior.
I rewrote handling player objects now to not depend one the object name being "".